data/transactions/logic/TEAL_opcodes.md

```diff
@@ -727,3 +741,110 @@ params: account index, asset id. Return: did_exist flag (1 if exist and 0 otherw
 
 
 params: txn.ForeignAssets offset. Return: did_exist flag (1 if exist and 0 otherwise), value.
+
+## assert
+
+- Opcode: 0x72
+- Pops: *... stack*, uint64
+- Pushes: _None_
+- immediately fail unless value X is a non-zero number
+- LogicSigVersion >= 3
+
+## min_balance
+
+- Opcode: 0x73
+- Pops: *... stack*, uint64
+- Pushes: uint64
+- get minimum balance for the requested account specified by Txn.Accounts[A] in microalgos. A is specified as an account index in the Accounts field of the ApplicationCall transaction, zero index means the sender
+- LogicSigVersion >= 3
+- Mode: Application
+
+## getbit
+
+- Opcode: 0x74
+- Pops: *... stack*, {any A}, {uint64 B}
+- Pushes: uint64
+- pop an index A, a target B (integer or byte-array). Pushes the Ath bit of B.
+- LogicSigVersion >= 3
+
+## setbit
+
+- Opcode: 0x75
+- Pops: *... stack*, {any A}, {uint64 B}, {uint64 C}
+- Pushes: uint64
+- pop a bit A, index B, and target C. Sets the Bth bit of C to A, and push the result
+- LogicSigVersion >= 3
+
+## getbyte
+
+- Opcode: 0x76
+- Pops: *... stack*, {[]byte A}, {uint64 B}
+- Pushes: uint64
+- pop an integer A and byte-array B. Extract the Ath byte of B and push it as an integer
+- LogicSigVersion >= 3
+
+## setbyte
+
+- Opcode: 0x77
+- Pops: *... stack*, {[]byte A}, {uint64 B}, {uint64 C}
+- Pushes: []byte
+- pop a small integer A (between 0..255), and integer B, and byte-array C. Set the Bth byte of C to A, and push the result
+- LogicSigVersion >= 3
+
+## swap
+
+- Opcode: 0x78
+- Pops: *... stack*, {any A}, {any B}
+- Pushes: *... stack*, any, any
+- swaps two last values on stack: A, B -> B, A
+- LogicSigVersion >= 3
+
+## select
+
+- Opcode: 0x79
+- Pops: *... stack*, {any A}, {any B}, {uint64 C}
+- Pushes: any
+- selects one of two values to retain: A, B, C -> A ? B : C
+- LogicSigVersion >= 3
+
+## dig
+
+- Opcode: 0x80 {uint8 depth}
+- Pops: *... stack*, any
+- Pushes: *... stack*, any, any
+- duplicate value N from the top of stack
+- LogicSigVersion >= 3
+
+## stxn
+
+- Opcode: 0x81 {uint8 transaction field index}
+- Pops: *... stack*, uint64
+- Pushes: any
+- push field to the stack from transaction A in the current group
+- LogicSigVersion >= 3
+
+for notes on transaction fields available, see `txn`. If top of stack is _i_, `stxn field` is equivalent to `gtxn _i_ field`.
+
+## stxna
+
+- Opcode: 0x82 {uint8 transaction field index}{uint8 transaction field array index}
+- Pops: *... stack*, uint64
+- Pushes: any
+- push value from an array field from transaction A in the current group
+- LogicSigVersion >= 3
+
+## pushbytes
+
+- Opcode: 0x83
```

Comment:
note the immediate data `{varuint length}{[length]byte data}`

Reply:
Thanks, got it